packages/generate-plugin/templates/src/_index.js.tpl

```diff
@@ -1,16 +1,16 @@
 /**
  * See the webpack docs for more information about plugins:
- * https://github.com/webpack/docs/wiki/how-to-write-a-plugin
+ * https://webpack.js.org/contribute/writing-a-plugin/#basic-plugin-architecture
  */
 
-function <%= name %>(options) {
-	// Setup the plugin instance with options...
+class <%= name %> {
+  apply(compiler) {
+    compiler.hooks.done.tap('Hello World Plugin', (
+      stats /* stats is passed as argument when done hook is tapped.  */
```

Comment:
```suggestion
      stats /* stats is passed as an argument when done hook is tapped.  */
```

Reply:
@jamesgeorge007 take reference from documentation

Reply:
The former isn't grammatically valid.